docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -73,3 +73,18 @@ and may interfere with indexing, search, and other activities in your cluster.
 Do not increase this setting without carefully verifying that your cluster has
 the resources available to handle the extra load that will result.
 
+ifdef::permanently-unreleased-branch[]
```

Comment:
Related to the comment about the feature flag, I don't think we need this guard in the docs.